lib/evss/base_service.rb

```diff
@@ -56,7 +56,8 @@ def conn
         faraday.use      Faraday::Response::RaiseError
         faraday.response :snakecase, symbolize: false
         faraday.response :json
-        faraday.adapter  :httpclient
+        faraday.use(Common::Client::Middleware::Request::RemoveCookies)
```

Comment:
minor nit - is there a reason to not use the existing formatting for the middleware and not grouping it with the existing middleware?

Reply:
i can change it to the existing formatting, this middleware must be added right before http_client because otherwise the value of `@app` will be different

Reply:
to get a symbolized ref you can register the middleware through `Faraday::Middleware.register_middleware` e.g.
https://github.com/department-of-veterans-affairs/breakers/blob/master/lib/breakers.rb#L12